roles/taca/templates/site_taca_sarek_delivery.yml.j2

```diff
@@ -68,3 +57,64 @@ deliver:
             - {{ ngi_resources }}/TACA/apply_recalibration.sh
             - <STAGINGPATH>/01-Resources/
             - required: True
+{% endif %}
+{% if "sthlm" == site %}
+    datapath: <ROOTPATH>/DATA/<PROJECTID>
+    stagingpathhard: <ROOTPATH>/DELIVERY_HARD/<PROJECTID>
+    files_to_deliver:
+        -
+            - <DATAPATH>/<SAMPLEID>/*/*
+            - <STAGINGPATH>/<SAMPLEID>/02-FASTQ
+            - required: True
+        -
+            - <ANALYSISPATH>/<SAMPLEID>/SarekGermlineAnalysis/results/Annotation/*
+            - <STAGINGPATH>/<SAMPLEID>/01-SarekGermline-Results/Annotation/
+            - required: True
+        -
+            - <ANALYSISPATH>/<SAMPLEID>/SarekGermlineAnalysis/results/pipeline_info/results_description.html
+            - <STAGINGPATH>/<SAMPLEID>/01-SarekGermline-Results/pipeline_info
+            - required: True
+        -
+            - <ANALYSISPATH>/<SAMPLEID>/SarekGermlineAnalysis/results/pipeline_info/software_versions.csv
+            - <STAGINGPATH>/<SAMPLEID>/01-SarekGermline-Results/pipeline_info
+            - required: True
+        -
+            - <ANALYSISPATH>/<SAMPLEID>/SarekGermlineAnalysis/results/Preprocessing/TSV/duplicates_marked*<SAMPLEID>.tsv*
+            - <STAGINGPATH>/<SAMPLEID>/01-SarekGermline-Results/Preprocessing/TSV
+            - required: True
+        -
+            - <ANALYSISPATH>/<SAMPLEID>/SarekGermlineAnalysis/results/Preprocessing/<SAMPLEID>/DuplicatesMarked/*
+            - <STAGINGPATH>/<SAMPLEID>/01-SarekGermline-Results/Preprocessing/DuplicatesMarked
+            - required: True
+        -
+            - <ANALYSISPATH>/<SAMPLEID>/SarekGermlineAnalysis/results/Preprocessing/<SAMPLEID>/Recalibrated/*
+            - <STAGINGPATH>/<SAMPLEID>/01-SarekGermline-Results/Preprocessing/Recalibrated
+            - required: True
+        -
+            - <ANALYSISPATH>/<SAMPLEID>/SarekGermlineAnalysis/results/Reports/*
+            - <STAGINGPATH>/<SAMPLEID>/01-SarekGermline-Results/Reports
+            - required: True
+        -
+            - <ANALYSISPATH>/<SAMPLEID>/SarekGermlineAnalysis/results/VariantCalling/<SAMPLEID>/*
+            - <STAGINGPATH>/<SAMPLEID>/01-SarekGermline-Results/VariantCalling/
+            - required: True
+    misc_files_to_deliver:
+        -
+            - {{ ngi_site_softlinks }}/ACKNOWLEDGEMENTS.txt
+            - <STAGINGPATH>
+        -
+            - <ANALYSISPATH>/reports/*
+            - <STAGINGPATH>/00-Reports
+        -
+            - <ANALYSISPATH>/*multiqc_report.html
+            - <STAGINGPATH>/00-Reports
+        -
+            - <ANALYSISPATH>/sarek_multiqc_report/*multiqc_report.html
+            - <STAGINGPATH>/00-Reports
+        -
+            - {{ ngi_resources }}/TACA/{{ site }}/DELIVERY.README.SAREK.txt
+            - <STAGINGPATH>
+            - required: True
+    save_meta_info: True
+    xmlgen_ignore_lib_prep: False
```

Comment:
`xmlgen_ignore_lib_prep` isnt needed anymore and can be removed.

Reply:
Thanks, I'll remove it.